src/slack.js

```diff
@@ -376,19 +296,13 @@ async function handleCSVUpload(event, bot, say) {
   }
 }
 
-async function showError(state, channel, msg, ts = null) {
+async function showError(channel, msg, ts = null) {
   await boltApp.client.chat[ts ? 'update' : 'postMessage']({
     channel,
     ts,
     as_user: true,
     blocks: [
-      {
-        type: 'section',
-        text: {
-          type: 'mrkdwn',
-          text: `:exclamation: ${msg}`,
-        },
-      },
+      sectionBlock('mrkdwn', `:exclamation: ${msg}`),
     ],
   })
 }
```

Comment:
I really like we got from 394 lines in this file to 308 - much more readable now 👍